api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -8351,10 +8351,10 @@ public CohortDefinition getMQ9Num(int flag) {
 
     if (flag == 1 || flag == 3) {
       cd.setCompositionString(
-          "A AND requestCd4 AND NOT (C OR D OR E OR pregnantOnPeriod OR breastfeedingOnPeriod) AND AGE");
+          "A AND D AND breastfeedingOnPeriod AND requestCd4 AND NOT (C OR E OR pregnantOnPeriod) AND AGE");
     } else if (flag == 2 || flag == 4) {
       cd.setCompositionString(
-          "A AND resultCd4 AND NOT (C OR D OR E OR pregnantOnPeriod OR breastfeedingOnPeriod) AND AGE");
+          "A AND breastfeedingOnPeriod AND D AND resultCd4 AND NOT (C OR E OR pregnantOnPeriod) AND AGE");
```

Comment:
This changes should not be applied on the numerator, please revert the changes and apply accordingly